sources/web/datalab/common.d.ts

```diff
@@ -14,6 +14,14 @@
 
 declare module common {
 
+  interface UserSettings {
```

Comment:
One of the things I have found a little confusing is the use of the term "settings" to refer to both application configuration settings and user preferences. Do you suppose we could start calling those "app config" and "user preferences"? I feel like those terms are the more commonly used terms for these two types of information.